tests/src/image-smoke.spec.ts

```diff
@@ -120,4 +114,33 @@ describe('Image workflow verification', async () => {
     imagesPage = await imageDetailsPage.deleteImage();
     expect(await imagesPage.waitForImageDelete('build-test-image')).toBeTruthy();
   });
+
+  test('Prune images', async () => {
+    const imageList = ['quay.io/podman/image1', 'quay.io/podman/image2'];
```

Comment:
I would prefer for this to be declared outside of the test case at the top of the smoke test

Reply:
ok, but is there a specific reason?

Reply:
To be honest I just looked at the other tests inside the test file and all of them have the image names defined inside the test, not globally.

Reply:
Well, turning things like these into a const readonly at the top of the test file improves readability, you know right-away what image is being used, constants are a good practice in general

Reply:
Moved the image names to global context for all tests in the file.